README.md

```diff
@@ -369,10 +369,10 @@ credentials:
 unclassified:
   gitLabServers:
     servers:
-      - credentialsId: "i<3GitLab"
+      - checkoutCredentialsId: "i<3GitLab"
         manageHooks: true
         name: "gitlab.com"
-        serverUrl: "https://gitlab.com"
+        serverName: "https://gitlab.com"
```

Comment:
name and serverName are super confusing and conflicting. :cry:

Reply:
I have fixed it.